src/utils/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { ListProps } from '../../types/Todo';
+
+export const TodoList: React.FC<ListProps> = ({
+  visibleTodos,
+  handleTodoStatusChange,
+  handleDeleteTodo,
+  loadingTodoIds,
+  tempTodo,
+  updateTodo,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {visibleTodos.map(todo => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          handleTodoStatusChange={handleTodoStatusChange}
+          handleDeleteTodo={handleDeleteTodo}
+          loadingTodoIds={loadingTodoIds}
+          updateTodo={updateTodo}
+        />
+      ))}
+
+      {tempTodo && (
+        <div
```

Comment:
you duplicate part of code here
You should just adjust your `TodoItem` component to be able to use it for tempTodo

Reply:
it was the idea of my mentor to make it in this way.

Reply:
hm, sounds strange, can you explain the mentor's idea?

Reply:
I don't remember, what was the purpose of this feature, but it 100% not my idea.